features/frontend/products.feature

```diff
@@ -1,73 +1,80 @@
+    And channel "WEB-EU" has following configuration:
+      | taxonomy |
+      | Category |
+    And channel "WEB-EU" has following products assigned:
+      | product         |
+      | Super T-Shirt   |
+      | Symfony T-Shirt |
+    And channel "WEB-US" has following products assigned:
+      | product          |
+      | Sylius Tee       |
+      | Black T-Shirt    |
+      | Doctrine T-Shirt |
+    And there are following reviews:
+      | title       | rating | comment               | author      | product         | subject type |
+      | Lorem ipsum | 5      | Lorem ipsum dolor sit | bar@foo.com | Symfony T-Shirt | product      |
 
-    Scenario: Browsing products by taxon
-        Given I am on the store homepage
-        When I follow "T-Shirts"
-        Then I should see there 1 products
-        And I should see "Super T-Shirt"
+  Scenario: Browsing products by taxon
+    Given I am on the store homepage
+    When I follow "T-Shirts"
+    Then I should see there 1 products
+    And I should see "Super T-Shirt"
 
-    Scenario: Empty index of products
-        Given there are no products
-        And I am on the store homepage
-        When I follow "Gloves"
-        Then I should see "There are no products to display"
+  Scenario: Empty index of products
+    Given there are no products
+    And I am on the store homepage
+    When I follow "Gloves"
+    Then I should see "There are no products to display"
 
-    Scenario: Accessing product page via title
-        Given I am on the store homepage
-        And I follow "PHP T-Shirts"
-        When I click "Symfony T-Shirt"
-        Then I should be on the product page for "Symfony T-Shirt"
+  Scenario: Accessing product page via title
+    Given I am on the store homepage
+    And I follow "PHP T-Shirts"
+    When I click "Symfony T-Shirt"
+    Then I should be on the product page for "Symfony T-Shirt"
 
-    Scenario: Display only products for current channel
-        Given I am on the store homepage
-        Then I should see "Super T-Shirt"
-        But I should not see "Black T-Shirt"
+  Scenario: Display only products for current channel
+    Given I am on the store homepage
+    Then I should see "Super T-Shirt"
+    But I should not see "Black T-Shirt"
 
-    Scenario: Display proper product price for specific channel
-        Given I am on the store homepage
-        Then I should see "Super T-shirt"
-        And I should see "€15.99"
+  Scenario: Display proper product price for specific channel
+    Given I am on the store homepage
+    Then I should see "Super T-shirt"
+    And I should see "€15.99"
 
-    Scenario: Display product attributes
-        Given I am on the product page for "Super T-shirt"
-        Then I should see "Wool"
+  Scenario: Display product attributes
+    Given I am on the product page for "Super T-shirt"
+    Then I should see "Wool"
```

Comment:
If we do anything with that it will be done in https://github.com/Sylius/Sylius/pull/4006, so please do not change it in this PR.